### DISCUSSION TITLE
Metrics and its formats

### DISCUSSION THREAD

**Discussion by ashwinrajanta** [GoogleChrome/lighthouse#15372] [General] [answered]
what is this values
"observedLastVisualChangeTs": 18549157723,
"observedLoadTs": 18536757914,
"observedTraceEndTs": 18549160231,

How do i convert it to human readable format??
is it UTC or epoch or what format?

is there any documents clearly mentioned for each metrcis and what it states???

**Comment by adamraine** [comment, marked as answer]
> How do i convert it to human readable format??

There should be an `observedTimeOriginTs` entry which represents the start of the page navigation. You can do `(observedLoadTs - observedTimeOriginTs) / 1000` to get the load time in milliseconds. Please keep in mind that this value will be *unthrottled* if you are using the default Lighthouse throttling settings (See https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md for more info)

> is it UTC or epoch or what format?

I believe it's microseconds since system boot, although this needs confirmation.

> is there any documents clearly mentioned for each metrcis and what it states???

No, this is a hidden audit for debug purposes.
